docs/index.md

```diff
@@ -9,31 +9,37 @@ These dbt-core features include:
 3. __[Access](https://docs.getdbt.com/docs/collaborate/govern/model-access)__ - control the `access` level of models within groups
 4. __[Versions](https://docs.getdbt.com/docs/collaborate/govern/model-versions)__ - create and increment versions of particular models.
 
+Additionally, `dbt-meshify` automates the code development required to split a monolithic dbt project into component projects, or connect multiple pre-existing dbt projects using cross-project `ref`.
+
 This package leverages the dbt-core Python API to allow users to use standard dbt selection syntax for each of the commands in this package (unless otherwise noted). See details on each of the specific commands available on the [commands page](commands.md)
 
-## Basic Usage
+## Getting Started
+
+This package helps automate the code development required for adding the dbt-core model governance features mentioned above. 
+
+The first question to ask yourself is "which of these features do I want to add to my project"? Do you want to add contracts, create a new group, split your monolithic dbt project in two? Your answer to this question will establish which `dbt-meshify` command is right for you! 
+
+This package consists of **component** and **global** commands - so you can decide how to best break apart your work. 
 
-Each of the available [commands page](commands.md) allows you to add one (or many) of the above features to a set of models specified by the selection syntax in the command.
+The **component** commands allow you to do a single step at a time and begin with `dbt-meshify operation`. For example, if you wanted to add a new version to a model, you would run something like `dbt-meshify operation add-version --select fct_orders`. This command would:
+1. add a new version to `fct_orders`
 
-The goal of this package is to make it more straightforward to apply to your project so that splitting apart a monolithic project into component projects is a more automated, dbt-tonic experience.
+and that's it!
 
-The process of splitting a dbt monolith apart roughly requires you to:
+The **global** commands combine _multiple_ **component** commands to complete a larger set of work and begin with `dbt-meshify`. For example, if you wanted to define a group for a subset of your models, you would run something like `dbt-meshify group finance --owner-name "Monopoly Man" --select +tag:finance`. This command would:
+1. define a new group named "finance" in your dbt project, setting the owner name to "Monopoly Man"
+2. add all models tagged with "finance" to that new group
+3. set `access` to protected for all "leaf" models (models with no downstream dependencies) and models with cross-group dependencies
+4. add contracts to all public nodes ... TO DO: if access is "protected" instead of "public" should we add contracts? would need to update examples below as well
```

Comment:
TO DO note here - if access is "protected" instead of "public" should we add contracts? would need to update examples below as well

Reply:
I think so! IMHO, if a model is not `private` and it is in a group, then it is liable to be an interface to another model => Could benefit from having a contract.

Reply:
Ok -  i'm updating this to "add contracts to all 'protected' nodes" since the group command doesn't currently make and nodes "public"